pages/common/git-repl.md

```diff
@@ -0,0 +1,21 @@
+# git repl
+
+> Open a Git shell-like command line.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-repl>.
+
+- Open a Git shell-like command line:
+
+`git repl`
+
+- Run a git command whilst in REPL:
```

Comment:
```suggestion
- Run a Git subcommand whilst in the Git interactive shell:
```

Reply:
```suggestion
- Run a Git command whilst in the Git interactive shell:
```

Reply:
`Whilst` sounds a bit lofty for our cirumstances. How about just using `while`?

Reply:
I don't love the use of the word myself 🤷🏻‍♂️

Reply:
Both are correct whilst `whilst` is a little more formal

Reply:
Almost a bit too formal for a documentation.

Reply:
Ok, it has been changed 
https://github.com/tldr-pages/tldr/pull/5974/commits/53542c30746e786f72f2e55d6e72f357ad1a12bc

Reply:
Also this has been fixed, it will be in my next commit, which is just being waited to be pushed